_includes/files.html

```diff
@@ -7,22 +7,44 @@
   displayed inside a pre block and include a download link underneath.
 
   @param {string} "title" - the yml meta field for title in the download file
+  @param {string} "language" - the coding language used for syntax highlighting
 
 -->
 {% endcomment %}
 
+{% comment %}
+<!-- determine correct collection based on language param -->
+{% endcomment %}
+{% if include.language == "yml" or include.language == "yaml" %}
+	{% assign files = site.ymls | where: 'title', include.title %}
+{% else %}
+	{% assign files = site.files | where: 'title', include.title %}
+{% endif %}
+
+{% comment %}
+<!-- determine syntax highlighting language.  Defaults to bash -->
```

Comment:
Nice catch, you are exactly correct, the code is doing the correct thing.  My first attempt I did default it to bash, thus the comment, but after re-reading the issue description before submitting the PR I noticed I hadn't done that correct, so fixed the code, but forgot to update the comment.  Comment is updated in the newest commit now! 9dfb3b7322ce160cd280217d3059a8fcea35e59e